unified_planning/grpc/proto_reader.py

```diff
@@ -912,12 +912,16 @@ def _convert_compiler_result(
                 original_action_instance.action,
                 original_action_instance.actual_parameters,
             )
+        engine_metrics = None
+        if bool(result.metrics):
```

Comment:
```suggestion
        if result.metrics:
```

Aren't those 2 versions the same?